erpnext/accounts/doctype/sales_invoice/sales_invoice.json

```diff
@@ -615,12 +564,11 @@
   }, 
   {
    "allow_bulk_edit": 0, 
-   "allow_in_quick_entry": 0, 
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
    "columns": 0, 
-   "depends_on": "return_against", 
+   "depends_on": "is_return", 
```

Comment:
Why changed?
is_return field is editable now in standalone debit/credit note
return_against set from the backend when user creates the return entry against the sales invoice